Sources/Nitrozen-SwiftUI/Elements/EmptyView/NitrozenEmptyView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  NitrozenEmptyView.swift
+//  
+//
+//  Created by Keyur Patel on 20/04/23.
+//
+
+import SwiftUI
+
+public struct NitrozenEmptyView: View {
+    
+    var topView: AnyView?
```

Comment:
instead top view, use similar concept to CustomView enum that we have used at other places